src/lib/config.ts

```diff
@@ -989,7 +992,7 @@ export function getURL(url: string, target: string[]) {
                     k =>
                         url.match(k) &&
                         getDeepProperty(conf.subconfigs[k], target) !==
-                            undefined,
+                        undefined,
```

Comment:
Should probably just run prettier on this file to fix all these indentation issues.

Reply:
Will do.